tests/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 find_package(Boost 1.55.0 REQUIRED)
 
+# We move the test files into the testing directory
+file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/hypervolume_test_data/" DESTINATION "${CMAKE_BINARY_DIR}/tests/hypervolume_test_data/")
```

Comment:
I do not like the trick very much, for cec2013 is ok as it gives the user some advantage, but these are for testing. I think for this purpose it better to have files than to add big static vectors that end up being parsed by clang, doxygen and more ....